config/deploy.rb

```diff
@@ -43,5 +43,12 @@
       execute "cd #{current_path} && ./gradlew jar"
     end
   end
+
+  task :install_dist do
+    on roles(:app) do
+      execute "cd #{current_path} && ./gradlew build installDist"
+    end
+  end
 end
 after 'deploy:finished', 'gradle:jar'
+after 'deploy:finished', 'gradle:install_dist'
```

Comment:
Kill it - we can easily add it back later if we need it.  Seems like ... we won't -- b/c we'll use installDist instead.  :-)